airbyte-webapp/src/components/ui/Icon/index.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import { Icon } from "./Icon";
+
+export default {
+  title: "Ui/Icon",
+  component: Icon,
+} as ComponentMeta<typeof Icon>;
+
+const Template: ComponentStory<typeof Icon> = (args) => <Icon {...args} />;
+
+export const ArrowRightIcon = Template.bind({});
```

Comment:
I think we should just have one story for `Icon` (instead of having an individual story per specific `type` property here), since that's really the component a dev would consume in their components, and just have a control in the story for the type, so it reflects more how a dev actually will use that component later on.